src/Compatibility/Core/src/Compatibility.csproj

```diff
@@ -8,7 +8,7 @@
     <iOSRoot>iOS\</iOSRoot>
     <WindowsRoot>Windows\</WindowsRoot>
     <TizenRoot>Tizen\</TizenRoot>
-    <IsTrimmable Condition="!$(TargetFramework.StartsWith('netstandard'))">true</IsTrimmable>
+    <IsTrimmable>false</IsTrimmable>
```

Comment:
We shouldn't do this, as it will significantly increase app sizes. What was breaking?